src/components/SearchBar.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { useInput } from '../utilities/customHooks';
+import { validate } from '../utilities/helperFunctions';
```

Comment:
Really like the addition of the utility folder, helps with organization moving forward.

Reply:
Thank you!  I hope it'll help.